server/api/server.go

```diff
@@ -100,18 +105,41 @@ func (s *Server) Start(cfg config.Config) error {
 		return err
 	}
 
-	s.registerGRPCGatewayRoutes()
 	s.listener = listener
-	var h http.Handler = s.Router
-
 	s.mtx.Unlock()
 
+	// configure grpc-web server
+	if cfg.GRPC.Enable && cfg.GRPCWeb.Enable {
+		var options []grpcweb.Option
+		if cfg.API.EnableUnsafeCORS {
+			options = append(options,
+				grpcweb.WithOriginFunc(func(origin string) bool {
+					return true
```

Comment:
Do we want this to be hardcoded?

Reply:
I have kept the same behavior as before. The API constructor will look a bit weird if we add grpcweb options.